### DISCUSSION TITLE
[Ideas] Support Aggregation pushdown while using outer joins

### DISCUSSION THREAD

**Discussion by foreyes** [apache/cloudberry#561] [Ideas / Feature Requests]
### Description

Support aggregation pushdown while using outer joins

### Use case/motivation

Currently, CBDB don't support using aggregation pushdown when there is outer join in the query.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

**Comment by avamingli** [comment]
Any links to introduce the theory of outer join push down? Does PG UPSTREAM support it?
